test/lib/utils/format-bytes.js

```diff
@@ -0,0 +1,25 @@
+const t = require('tap')
```

Comment:
We tend not to do unit testing like this.  It encourages dead code that we only actually hit in tests, and not from the surface area of the code.  The existing `view` or `tar` tests should be where this is covered.